Ubuntu_14.04/job/header.sh

```diff
@@ -46,33 +46,55 @@ before_exit() {
 }
```

Comment:
ShellCheck has a few warnings for this file. Let's fix them.

Reply:
i am not fixing the `is_success` warning since i will be removing it in the next pr.